pruebas/scripts/seeker_listener.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+import rospy
+from sensor_msgs.msg import Image
+from darknet_ros_msgs.msg import BoundingBoxes
+from std_msgs.msg import Header
+from std_msgs.msg import String
+
+def callback(data):
+    for box in data.bounding_boxes:
+        rospy.loginfo(
+            "I found: {} ".format(box.Class)
+        )
```

Comment:
This is printing an object per line and it's not clear when we are detecting various objects simultaneously. 

I suggest something like
```suggestion
def callback(data):
    rospy.loginfo(f'I found: {", ".join([box.Class for box in data.bounding_boxes])}')
```
that prints 
`[INFO] [1628782231.061815, 236.987000]: I found: person, person, person`

Reply:
That is a "f-string", but I'm not sure if we should rely on them since they are only available since Python 3.6 https://realpython.com/python-f-strings/